modules/home/assets/js/components/home-screen.js

```diff
@@ -17,9 +18,10 @@ const HomeScreen = ( props ) => {
 					createNewPageUrl={ props.homeScreenData.create_new_page_url }
 				/>
 				<Box sx={ { display: 'flex', flexDirection: { xs: 'column', sm: 'row' }, justifyContent: 'space-between', gap: 3 } }>
-					<Box sx={ { flex: 1 } }>
+					<Container maxWidth="md" disableGutters={ true } sx={ { display: 'flex', flexDirection: 'column', gap: 3 } }>
```

Comment:
Why do we have a `<Container>` here?
Can't we add the flex styling inside the `<Box>` component?
Or could we use the `<Stack>` instead?

I don't like seeing containers with a maxWidth if we don't really need that.